docs/migration_2_to_3.rst

```diff
@@ -149,3 +149,16 @@ Telegram API Server
 
 - The `server` parameter has been moved from the `Bot` instance to `api` in `BaseSession`.
 - The constant `aiogram.bot.api.TELEGRAM_PRODUCTION` has been moved to `aiogram.client.telegram.PRODUCTION`.
+
+
+Telegram objects transformation (to dict, to json, from json)
+===========================================
+
+- Methods :class:`TelegramObject.to_object()`, :class:`TelegramObject.to_json()` and :class:`TelegramObject.to_python()` 
+  have been removed due to the use of `pydantic <https://docs.pydantic.dev/>`_ models.
+- :class:`TelegramObject.model_validate_json()` should be used instead of :class:`TelegramObject.to_object()` 
```

Comment:
Since this is a migration guide, it would be more convenient to indicate what needs to be changed to what. 
For example: method ___ should be replaced by ___

Reply:
fixed